### ISSUE TITLE
Keyboard focus is not visible on "Path", "Details" and "WS1138: UseHttpOnlyCookieAttribute" controls in high contrast mode.

### ISSUE THREAD

**Issue by shreyashi227** [microsoft/sarif-web-component#28]
### Actual:
Keyboard focus is not visible on "Path", "Details" and "WS1138: UseHttpOnlyCookieAttribute" controls in high contrast black/white mode using Tab key

### Expected:
Keyboard focus should be clearly visible on"Path", "Details" and "WS1138: UseHttpOnlyCookieAttribute" controls in high contrast black/white mode using Tab key

**Comment by jeffersonking** [comment]
"Path" and "Details" are fixed in v0.5 (2413b17b8cf2f5a11cf4238df5418f00eddd43a1). I see them with a cyan border. "WS1138: UseHttpOnlyCookieAttribute" I see with a blue border.

Note, we use the Formula Design System for our Tree. The behavior matching what I see on their [examples page](https://developer.microsoft.com/en-us/azure-devops/components/tree). If you find that the blue border of sufficient contrast, you may need to open an issue with them. They do not have a public GitHub repo where you can file issues. You may need to contact them internally at Microsoft.
